From 38d91d8750a882643caade6b1f6fd852bfa79199 Mon Sep 17 00:00:00 2001
From: Peter Swain <pswain@marvell.com>
Date: Tue, 20 Aug 2019 17:24:56 -0700
Subject: [PATCH 0425/1921] mmc: cavium: calibrate otx2 just once

There's no need to re-run the calibration,
it's for characterizing chip, not tracking variation
due to voltage/temperature drift, which execute_tuning
handles quite well with finer granularity

Also adds/clarifies debug logs

Change-Id: I46d5d4c30d126bfc6ba00c49cfd81fb143948f20
Signed-off-by: Peter Swain <pswain@marvell.com>
Reviewed-on: https://sj1git1.cavium.com/14550
Tested-by: sa_ip-sw-jenkins <sa_ip-sw-jenkins@marvell.com>
Reviewed-by: Sunil Kovvuri Goutham <Sunil.Goutham@cavium.com>
Reviewed-on: https://sj1git1.cavium.com/16106
Tested-by: Sunil Kovvuri Goutham <Sunil.Goutham@cavium.com>
Reviewed-on: https://sj1git1.cavium.com/c/IP/SW/kernel/linux/+/26920
Reviewed-by: Chandrakala Chavva <cchavva@marvell.com>
[WK: The original patch got from Marvell sdk11.21.09]
Signed-off-by: Wenlin Kang <wenlin.kang@windriver.com>
---
 drivers/mmc/host/cavium.h | 1 -
 1 file changed, 1 deletion(-)

diff --git a/drivers/mmc/host/cavium.h b/drivers/mmc/host/cavium.h
index 9c9f663cceb5..f38353171678 100644
--- a/drivers/mmc/host/cavium.h
+++ b/drivers/mmc/host/cavium.h
@@ -290,7 +290,6 @@ irqreturn_t cvm_mmc_interrupt(int irq, void *dev_id);
 int cvm_mmc_of_slot_probe(struct device *dev, struct cvm_mmc_host *host);
 int cvm_mmc_of_slot_remove(struct cvm_mmc_slot *slot);
 
-
 extern const char *cvm_mmc_irq_names[];
 
 static inline bool is_mmc_8xxx(struct cvm_mmc_host *host)
-- 
2.31.1

